platform/src/main/java/org/stellar/anchor/platform/component/share/ClientsBeans.java

```diff
@@ -3,13 +3,29 @@
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.stellar.anchor.api.exception.InvalidConfigException;
+import org.stellar.anchor.client.ClientService;
+import org.stellar.anchor.client.DefaultClientService;
+import org.stellar.anchor.config.ClientsConfig;
 import org.stellar.anchor.platform.config.PropertyClientsConfig;
+import org.stellar.anchor.platform.config.PropertyClientsConfig_DEPRECATED;
 
 @Configuration
 public class ClientsBeans {
   @Bean
   @ConfigurationProperties(prefix = "")
-  PropertyClientsConfig clientsConfig() {
+  PropertyClientsConfig_DEPRECATED clientsConfigDeprecated() {
+    return new PropertyClientsConfig_DEPRECATED();
+  }
+
+  @Bean
+  @ConfigurationProperties(prefix = "clients")
+  ClientsConfig clientsConfig() {
     return new PropertyClientsConfig();
   }
```

Comment:
One thing that annoys me about the asset config is that specifying it in-line looks redundant: 

```yaml
assets:
  type: yaml
  value:
    assets:
       - ...
```

Ideally, we make `type` optional and if it is not present, assume its yaml. So it would look like this:

```yaml
assets:
  - ...
```

or

```yaml
assets:
  type: file
  value: /path/to/clients.yaml
```

Reply:
I think this is the new setup, you only need to define like this:
```
clients.type=file
clients.value=/config/clients.yaml
```